trips_01_import.R

```diff
@@ -0,0 +1,124 @@
+#This file does:
+#import the csv trip data to data frames. Then combine trips data by the year. 2013 - 2017, 2021 to 2022. 
```

Comment:
I'd drop the "This file does:" line all together start with "Imports the CSV trip data into data frames...."